fdbrpc/Locality.h

```diff
@@ -157,6 +157,32 @@ struct LocalityData {
 		_data[key] = value;
 	}
 
+	int size() const { return _data.size(); }
+	std::pair<Standalone<StringRef>, Optional<Standalone<StringRef>>> getItem(int i) {
```

Comment:
I had a comment about this on the other PR, but I think using a `begin` and `end` would actually be simpler code-wise and wouldn't imply some property (i.e. efficient random access) that doesn't hold.